src/main/java/com/google/codeu/data/User.java

```diff
@@ -0,0 +1,21 @@
+package com.google.codeu.data;
+//import java.util.UUID;
```

Comment:
It's generally not good practice to leave commented-out code in code that gets deployed, but not a big deal.